assets/pages/contributions-landing/contributionsLandingUK.jsx

```diff
@@ -0,0 +1,87 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+
+import SimpleHeader from 'components/headers/simpleHeader/simpleHeader';
+import SimpleFooter from 'components/footers/simpleFooter/simpleFooter';
+import IntroductionText from 'components/introductionText/introductionText';
+
+import pageStartup from 'helpers/pageStartup';
+import getQueryParameter from 'helpers/url';
+
+import reducer from './reducers/reducers';
+import ContributionsBundle from './components/contributionsBundle';
+
+
+// ----- Page Startup ----- //
+
+pageStartup.start();
+
+
+// ----- Redux Store ----- //
+
+const store = createStore(reducer, {
+  intCmp: getQueryParameter('INTCMP', 'gdnwb_copts_bundles_landing_default'),
+});
+
+
+// ----- Copy ----- //
+
+const introductionCopy = [
```

Comment:
Ah, nope it's intentionally duplicated. I have a suspicion that this copy will get played around with on different pages, it just happens to be the same at the moment because we haven't had too much design on this page.